reactor-core/src/test/java/reactor/core/scheduler/SchedulersMetricsTest.java

```diff
@@ -256,4 +258,27 @@ public void shouldReportExecutionTimes(Supplier<Scheduler> schedulerSupplier, St
 					.isEqualTo(taskCount);
 		});
 	}
+
+	@Test
+	public void shouldRemoveOnShutdown() {
+		Scheduler scheduler = afterTest.autoDispose(Schedulers.newParallel("A", 1));
```

Comment:
to test the above, make it `2`, capture a `Worker` to hold onto an executor and then call `createWorker()` a second time then `dispose()` that worker.
you can then verify the registry finds half as much meters as it did before the dispose, and that the meters for the first executor are still there.